.changeset/cold-ties-guess.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Hey @kaydbwithers, you can remove this changeset file. The PR only includes changes to a Story, so we wouldn't necessarily want to release a new version. The Storybook deploy should pick this up just fine.

I don't think our CI requires a Changeset file to be present (yet). Otherwise, `yarn changeset --empty` creates an empty one (or delete this one's contents).

Reply:
Haha, just saw @HeartSquared also replied with the same ;)